integration_tests/integration/assess-recall.spec.js

```diff
@@ -193,20 +193,15 @@ context('Assess a recall', () => {
     cy.task('expectListRecalls', { expectedResults: [] })
     cy.task('expectSearchResults', { expectedSearchTerm: nomsNumber, expectedSearchResults: searchResponse })
 
-    cy.readFile('integration_tests/expected-revocation-order.pdf', 'base64').then(base64EncodedPdf => {
+    cy.readFile('integration_tests/test.pdf', 'base64').then(base64EncodedPdf => {
```

Comment:
Is the. file 'expected-revocation-order.pdf'. still needed?

Reply:
I replaced it with test.pdf